server/apps/immich/src/api-v1/asset/asset.module.ts

```diff
@@ -26,7 +26,6 @@ const ASSET_REPOSITORY_PROVIDER = {
     CommunicationModule,
     BackgroundTaskModule,
     DownloadModule,
-    AlbumModule,
```

Comment:
@alextran1502 I was getting a circular dependency error because of this.
Below we have `forwardRef(() => AlbumModule)` so I think we don't need this, do we? (I didn't test the app after this change)

Reply:
Yeah, that has been there for awhile and I've been meaning to get rid of it.

Reply:
I did check this and the app still compiles without it.

Reply:
> Looks good. I'm pretty sure the `TypeOrmModule` dependency can be removed, although I can't test it locally.

Yes you are right, that is not needed anymore